app/Http/Controllers/ThirdParty/CallPowerController.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Chompy\Http\Controllers\ThirdParty;
+
+use Illuminate\Http\Request;
+use Chompy\Services\Rogue;
+use Chompy\Http\Controllers\Controller;
+
+class CallPowerController extends Controller
+{
+
+    /**
+     * Create a controller instance.
+     *
+     * @return void
+     */
+    public function __construct(Rogue $rogue)
+    {
+      // @TODO: add role middleware?
```

Comment:
Do we need to add role middleware here or will the partner specific static API key be sufficient here?

Reply:
Yeah, we'd only use `role` middleware if we're getting an OAuth token (with a user or machine role attached). There's no real concept of a "role" when taking in a static key like this.